From c17e0cb12d2946cc9f72ca9df52c800542ef6490 Mon Sep 17 00:00:00 2001
From: Jai Luthra <jai.luthra@ideasonboard.com>
Date: Tue, 22 Sep 2026 12:10:54 +0530
Subject: libcamera: v4l2_subdevice: Drop Type::EmbeddedData

All generic raw formats use Type::Metadata, and the specific format or
layout used by the sensor for the embedded line is instead advertised
through the new V4L2_CID_METADATA_LAYOUT control.

So drop the EmbeddedData type from the enum as it is unused.

Signed-off-by: Jai Luthra <jai.luthra@ideasonboard.com>
(cherry picked from commit 9e3e005f003f230676d50416cdd003555b651998)
---
 include/libcamera/internal/v4l2_subdevice.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/libcamera/internal/v4l2_subdevice.h b/include/libcamera/internal/v4l2_subdevice.h
index 36848915f..5cec6a001 100644
--- a/include/libcamera/internal/v4l2_subdevice.h
+++ b/include/libcamera/internal/v4l2_subdevice.h
@@ -37,7 +37,6 @@ public:
 	enum class Type {
 		Image,
 		Metadata,
-		EmbeddedData,
 	};
 
 	bool isValid() const { return code != 0; }
